_sass/addon/module.scss

```
@@ -7,7 +7,7 @@
 %heading {
   color: var(--heading-color);
   font-weight: 400;
-  font-family: Lato, 'Microsoft Yahei', sans-serif;
+  font-family: $heading-font;
```

Update the variable name:

```suggestion
  font-family: $font-family-heading;
```